scripts/ci/install-minikube.sh

```diff
@@ -1,19 +1,18 @@
 #!/bin/bash
 
-set -v
-
 OS=linux
 ARCH=amd64
 TARGET_DIR="/usr/bin"
 
-MINIKUBE_VERSION="v0.24.1"
+
+MINIKUBE_VERSION="v0.25.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
 KUBECTL_VERSION="v1.9.0"
 KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
 
-[ -z "$MINIKUBE_FLAGS" ] && MINIKUBE_FLAGS="--extra-config=kubelet.CgroupDriver=systemd"
-[ -z "$MINIKUBE_VM_DRIVER" ] && MINIKUBE_VM_DRIVER=none
+export PATH=$PATH:$TARGET_DIR
+alias sudo='sudo -E'
```

Comment:
this is not really a good idea in non interactive mode, as you will forgot that sudo export environement variables